src/renderer/components/icon/flat_arrow.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Just wondering why we didn't use material icon arrows here.

Reply:
To fully match the wireframes we have.